test/common.py

```diff
@@ -2160,8 +2161,16 @@ def get_zlib_library(self, cmake):
 # Run a server and a web page. When a test runs, we tell the server about it,
 # which tells the web page, which then opens a window with the test. Doing
 # it this way then allows the page to close() itself when done.
-def harness_server_func(in_queue, out_queue, port):
+def harness_server_func(in_queue, out_queue, port, base_path):
   class TestServerHandler(SimpleHTTPRequestHandler):
+    # Convert the path relative to what the base_path is set to.
+    # TODO is this still needed anymore? Tests seem to run fine without it.
```

Comment:
I'm confused - the TODO suggests this is old code, which might not be needed. But this is new code adding in this PR? At least I can't see it as originating from any old code moved around in this diff.

Reply:
Hmm, interesting. Not sure what the reason for that was...

If tests run fine without this, I guess we can leave it out?